typescript/introduction/intro-typescript/hello-typescript.md

```diff
@@ -0,0 +1,69 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+links:
+  - '[ts-node](https://www.npmjs.com/package/ts-node){website}'
+
+---
+
+# Hello TypeScript
+
+---
+## Content
+
+We have promised you some TypeScript code, and here it is:
+
+```ts
+// hello.ts
+const hello = (name: string) => {
+  console.log(`Hello ${name}!`)
+}
+
+let myName = 'Andrei';
+hello(myName);
+```
+
+When running `tsc hello.ts` in the shell, a new file `hello.js` is created, and these are its contents:
+
+```js
+// hello.js
+var hello = function (name) {
+    console.log("Hello " + name + "!");
+};
+var myName = 'Andrei';
+hello(myName);
+```
+
+Depending on your application, this might be enough or you might want to run the `.js` file using node. In the latter case, instead of running two separate commands, you can install the [ts-node package](https://www.npmjs.com/package/ts-node) and let it do the work for you.
```

Comment:
```suggestion
Once you have the JavaScript code, you can run it wherever JavaScript can run (any Node.js or browser environment).

To convert TypeScript to JavaScript and run the generated JavaScript as a single step, we can use the [ts-node package](https://www.npmjs.com/package/ts-node).

```sh
# convert hello.ts to from
# Typescript to JavaScript
# and run the created JavaScript
ts-node hello.ts
```
```